docs/operations/faq/zone_aware_routing.rst

```diff
@@ -0,0 +1,53 @@
+.. _common_configuration_zone_aware_routing:
+
+Zone aware routing
+==================
+
+There are several steps required for enabling :ref:`zone aware routing <arch_overview_load_balancing_zone_aware_routing>`
+between source service A and destination service B. See below for specific configuration/setup on source
+and destination service/cluster.
+
+Envoy configuration on source service
+-------------------------------------
+This section describes specific configuration for Envoy running side by side with the service A.
+These are the requirements:
+
+* Envoy must be launched with :option:`--service-zone` option which defines availability zone for the current host.
```

Comment:
grammar (you are consistently missing "a", "the", etc. please read through and fix). There are problems elsewhere but I won't comment on them all now.